src/components/visualizations/chart/chartOptionsBuilder.ts

```diff
@@ -34,31 +43,23 @@ import {
     parseValue,
     stringifyChartTypes
 } from '../utils/common';
-import { getChartProperties } from './highcharts/helpers';
-import { unwrap } from '../../../helpers/utils';
+import { getComboChartOptions } from './chartOptions/comboChartOptions';
```

Comment:
seems suspicious that in this general file is imported something ComboChart-specific

Reply:
I see - it was always there... but if there's a chance to correct it, that would be nice